pymatgen/io/lobster/outputs.py

```diff
@@ -499,9 +490,9 @@ def __init__(
                 list_atom1=atoms1,
                 list_atom2=atoms2,
                 list_length=lens,
-                list_translation=translations,
+                list_translation=translations,  # type: ignore[arg-type]
                 list_num=nums,
-                list_icohp=icohps,
+                list_icohp=icohps,  # DEBUG: DanielYang: inconsistent type
```

Comment:
Need a closer look on this.

The `io.lobster.outputs.Icohplist` class:https://github.com/materialsproject/pymatgen/blob/c83d5d5280cf0004f7d3571b7afc41b7604d8124/pymatgen/io/lobster/outputs.py#L495-L507 `icohps` is `list[dict[Spin, float]]`:https://github.com/materialsproject/pymatgen/blob/c83d5d5280cf0004f7d3571b7afc41b7604d8124/pymatgen/io/lobster/outputs.py#L441-L447

While the `IcohpCollection` constructor expect `list[dict[Spin, icohpvalue]]`: https://github.com/materialsproject/pymatgen/blob/c83d5d5280cf0004f7d3571b7afc41b7604d8124/pymatgen/electronic_structure/cohp.py#L1069

And `IcohpValue` is not a superclass of `float`: https://github.com/materialsproject/pymatgen/blob/c83d5d5280cf0004f7d3571b7afc41b7604d8124/pymatgen/electronic_structure/cohp.py#L858